incubator/jaeger/templates/spark-cronjob.yaml

```diff
@@ -44,37 +45,19 @@ spec:
               value: {{ .Values.storage.type }}
             {{- if eq .Values.storage.type "cassandra" }}
             - name: CASSANDRA_CONTACT_POINTS
-              valueFrom:
-                configMapKeyRef:
-                  name: {{ include "jaeger.fullname" . }}
-                  key: cassandra.contact-points
+              value: {{ template "cassandra.contact_points" . }}
             - name: CASSANDRA_KEYSPACE
-              valueFrom:
-                configMapKeyRef:
-                  name: {{ include "jaeger.fullname" . }}
-                  key: cassandra.keyspace
+              value: {{ printf "%s_%s" "jaeger_v1" .Values.cassandra.config.dc_name | quote }}
               {{- end }}
             {{- if eq .Values.storage.type "elasticsearch" }}
             - name: ES_NODES
-              valueFrom:
-                configMapKeyRef:
-                  name: {{ include "jaeger.fullname" . }}
-                  key: es.server-urls
+              value: {{ template "elasticsearch.client.url" . }}
             - name: ES_NODES_WAN_ONLY
-              valueFrom:
-                configMapKeyRef:
-                  name: {{ include "jaeger.fullname" . }}
-                  key: es.nodes-wan-only
+              value: {{ .Values.storage.elasticsearch.nodesWanOnly | quote }}
             - name: ES_PASSWORD
-              valueFrom:
-                configMapKeyRef:
-                  name: {{ include "jaeger.fullname" . }}
-                  key: es.password
+              value: {{ .Values.storage.elasticsearch.password }}
```

Comment:
Imho passwords should be in a secret and used via secretKeyRef.

Reply:
Agreed. I will make an issue for this and tackle it in a separate PR. Thanks.

Reply:
https://github.com/helm/charts/issues/15122 created. I encourage you to submit a PR for this @monotek

Reply:
Agreed, #15122 created, let me try to include the fix in this PR.

Reply:
Changes applied.